utopia-remix/app/styles/contextMenu.css.ts

```diff
@@ -29,10 +29,9 @@ export const contextMenuDropdown = recipe({
     sprinkles({
       borderRadius: 'small',
       background: 'white',
-      color: 'lightModeBlack'
+      color: 'lightModeBlack',
```

Comment:
does it mean we don't yet run eslint / prettier for these files?

Reply:
I think this was untouched after we enabled prettier correctly